lib/extract_field/login_successful/StatusLoginSuccessfulParser.js

```diff
@@ -0,0 +1,17 @@
+const { ILoginSuccessfulParser } = require('./ILoginSuccessfulParser');
+
+class StatusLoginSuccessfulParser extends ILoginSuccessfulParser {
```

Comment:
is there a way with your implementation to configure that for one endpoint the success status will be for example 200 and for another a different value?

Reply:
This is a great comment since the login successful reporting method is decided for all endpoints -- so there are a few options.
1. If relevant, multiple status codes can be defined (200, 201, 202, etc.) but if credentials are extracted on any endpoint and one of these statuses is returned, it will be counted as login_successful = true.
2. It's possible to use a custom login successful parser wherein the customer can decide to use different logic depending on the endpoint.